src/main/resources/db/migration/V19__POL-650_add_org_id_columns.sql

```diff
@@ -1,15 +0,0 @@
-CREATE TABLE org_id_latest_update (
```

Comment:
Why is this here? This PR shouldn't contain any SQL file changes.

Reply:
It deletes the script. You wanted this to be deleted.

Reply:
reverted that one above and removed the other one